### ISSUE TITLE
Faezeh Rohani

### ISSUE THREAD

**Issue by Faezeh-Rohani-AI** [obophenotype/human-phenotype-ontology#11441]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@Faezeh-Rohani-AI  This issue is empty. Please submit a new issue if needed.
